conrad/__main__.py

```diff
@@ -1,16 +1,26 @@
 # -*- coding: utf-8 -*-
 
 from __future__ import absolute_import
-
-
+from __version__ import __version__
+import urllib.request, json
 __all__ = ("main",)
 
 
 def main():
     from conrad.cli import cli
 
     cli()
-
+def get_current_version():
+    url = "https://pypi.org/pypi/conference-radar/json"
+    with urllib.request.urlopen(url) as response:
+        data = json.loads(response.read().decode())
+    return data["info"]["version"]
 
 if __name__ == "__main__":
+    latest_version = get_current_version()
+
+    if __version__ < latest_version:
+        print("You are using conrad version " + __version__ + ", however, version " + latest_version + " is available.")
```

Comment:
Can you change it to a warning like it is done here: https://github.com/pypa/pip/blob/master/src/pip/_internal/self_outdated_check.py#L234